app/stylesheet/menu.scss

```diff
@@ -123,3 +140,18 @@
   // compensate for in-image padding
   margin: -4px 0 0 -8px;
 }
+
+.bx--side-nav__item {
```

Comment:
Is this living outside the menu? If not, can you move this inside the `#main-menu` to the same level as `.bx--side-nav.secondary`?

Reply:
Fixed it. please check updated code.